### PR TITLE
[REF] 8.0 information security manual

### PR DESCRIPTION
**Pull request by llacroix**
Depends on: 
- [x] #23

### CODE REVIEW

**Comment by bwrsandman** [comment]
Could you rename `information_security_manual.xml` to `document_page_data.xml` after the model

**Comment by bwrsandman** [comment]
Depends on #23

**Comment by max3903** [comment]
Tested on runbot.

:+1:

File: information_security_manual/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# -*- encoding: utf-8 -*-
```
Comment (dreispt) [inline information_security_manual/__init__.py]: s/encoding/coding

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**